src/Components/Card.js

```diff
@@ -0,0 +1,13 @@
+const Card = () => {
+   
+    
+
+    return (
+        <li className="Card">
+           <img src="https://rickandmortyapi.com/api/character/avatar/2.jpeg" alt="Profile picture" />
+           <p>Morty Smith</p>
+        </li>
```

Comment:
You're using here a `li` tag. That's good. But the `Card` component should be wrapped inside a `ul` tag in the `App.js`.